spec/azure.go

```diff
@@ -0,0 +1,5 @@
+package spec
+
+type Azure struct {
```

Comment:
For now there is just Location which is the Azure equivalent of AWS Region in the API.